pkg/monitoring/rules/alerts/vms.go

```diff
@@ -29,6 +29,18 @@ var (
 	fiftyMB = resource.MustParse("50Mi")
 
 	vmsAlerts = []promv1.Rule{
+		{
+			Alert: "VirtLauncherPodsStuckFailed",
+			Expr:  intstr.FromString("sum(kube_pod_status_phase{phase='Failed', pod=~'virt-launcher-.*'}) >= 200"),
+			For:   ptr.To(promv1.Duration("10m")),
+			Annotations: map[string]string{
+				"summary": "At least 200 virt-launcher pods are stuck in Failed state and not deleted for 10 minutes.",
+			},
+			Labels: map[string]string{
+				severityAlertLabelKey:        "critical",
+				operatorHealthImpactLabelKey: "critical",
```

Comment:
is this critical because of resource consumption?
seems harsh to instantly critically alert for this situation

maybe add a middle warning case for 100 pods or so

and add a description for why you consider each of the fields critical, doesnt look like the typical situation